### PR TITLE
NAS-118446 / 22.12 / add MISMATCH_VERSIONS to failover topbar

### PR DESCRIPTION
**Pull request by yocalebo**
Added this on backend here: https://github.com/truenas/middleware/pull/9937

### CODE REVIEW

**Review by denysbutenko — APPROVED** [review]
👍

**Comment by yocalebo** [comment]
Build failures seem unrelated to my changes. 🤷‍♂️

File: src/app/enums/failover-disabled-reason.enum.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export enum FailoverDisabledReason {
   NoLicense = 'NO_LICENSE',
   DisagreeVip = 'DISAGREE_VIP',
   MismatchDisks = 'MISMATCH_DISKS',
+  MismatchVersionss = 'MISMATCH_VERSIONS',
```
Comment (denysbutenko) [inline src/app/enums/failover-disabled-reason.enum.ts:10]: here is a typo that causes a failure

Comment (yocalebo) [reply src/app/enums/failover-disabled-reason.enum.ts:10]: Wow, nice!

**Pull request merged**